workitem.go

```diff
@@ -98,7 +101,11 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 // Create runs the create action.
 func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
-		wi, err := appl.WorkItems().Create(ctx.Context, ctx.Payload.Type, ctx.Payload.Fields)
+		currentUser, err := login.ContextIdentity(c.tokenManager, ctx)
```

Comment:
It generally works now, but not a fan of having to send the token.Manager around to every service that needs to read the user.